proto/api/v2/compliance_integration_service.proto

```diff
@@ -18,6 +18,7 @@ message ComplianceIntegration {
   // Collection of errors that occurred while trying to obtain compliance operator health info.
   repeated string status_errors = 6;
   bool operator_installed = 7;
+  string status = 8;
```

Comment:
Could this be an enum instead if we have a fixed set of statuses? Otherwise we have hardcoded strings in our code base, when having these strings as constants available it is easy to search for usages, renaming them and knowing which values are possible for the status field.

Reply:
+1  make this an enum